src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -785,7 +785,7 @@ class AccessibleBuffer extends DisposableStore {
 	) {
 		super();
 		this._focusContextKey = TerminalContextKeys.accessibleBufferFocus.bindTo(contextKeyService);
-		this._accessibleBuffer = this._terminal.element?.querySelector('.xterm-accessibility-buffer') as HTMLElement || undefined;
+		this._accessibleBuffer = this._terminal.element?.querySelector<HTMLElement>('.xterm-accessibility-buffer');
```

Comment:
Oh this will also cause a compile error I think, will need a change above:

```ts
_accessibleBuffer: HTMLElement | null | undefined
```

Even better:

```ts
this._accessibleBuffer = withNullAsUndefined(this._terminal.element?.querySelector<HTMLElement>('.xterm-accessibility-buffer'));
```